app.go

```diff
@@ -71,14 +87,15 @@ func (a *PuffApp) addOpenAPIRoutes() {
 	docsRouter.Get(".json", nil, func(c *Context) {
```

Comment:
why are we using this and not our JSONResponse, i probably made it like this but why